src/email-body.js

```diff
@@ -0,0 +1,66 @@
+const bodyStyle = `background: #60ADD6;
```

Comment:
@RhodesPeter have you asked @skibinska? We have walked through how to use handlebars server side without using the reply interface